linux companion compile error (XSDCXX_EXECUTABLE-NOTFOUND)

When compiling companion on Slackware linux got the error:
```
cmake ../src/
-- The C compiler identification is GNU 4.8.2
-- The CXX compiler identification is GNU 4.8.2
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Looking for XercesC 
--   found xerces-c: /usr/lib64/libxerces-c.so
-- Looking for Xsd 
-- XSDCXX_EXECUTABLE-NOTFOUND
CMake Error at CMakeLists.txt:27 (MESSAGE):
  XSD not found! Please set XSD path ...


-- Configuring incomplete, errors occurred!
```

Thing is I have XSD "All-In-One Package" package working in a non standard directory (no easy package for slackware, and I need it only for OpenTX).

The solution is quite simple, set XSDDIR to that directory! But it still produces the same problem... The search path in FindXsd.cmake does not include the sub-directories for the "All-In-One Package"!!!

Already solved, opening issue to reference in pull request! :wink:

p.s.
export XSDDIR=$DIR/xsd-3.3.0+dep/xsd

--- a/companion/src/cmake/FindXsd.cmake
+++ b/companion/src/cmake/FindXsd.cmake
@@ -14,7 +14,8 @@ FIND_PATH( XSD_INCLUDE_DIR xsd/cxx/parser/elements.hxx
   "C:/mingw/xsd-3.3.0-i686-windows/libxsd"
   $ENV{XSDDIR}/include
   $ENV{CODESYNTH}/include
-  /usr/local/include /usr/include  
+  /usr/local/include /usr/include
+  $ENV{XSDDIR}/libxsd
 )
 
 IF( WIN32 )
@@ -30,15 +31,16 @@ FIND_PROGRAM( XSDCXX_EXECUTABLE
   PATHS
     "/opt/local/xsd-3.3.0-i686-macosx/bin"
     "C:/Programs/xsd-3.3.0/bin"
-    "C:/mingw/xsd-3.3.0-i686-windows/bin"  
+    "C:/mingw/xsd-3.3.0-i686-windows/bin"
     "C:/Program Files/CodeSynthesis XSD 3.2/bin"
     $ENV{XSDDIR}/bin
     /usr/local/bin
     /usr/bin
+    $ENV{XSDDIR}/xsd
 )
 
 MESSAGE(STATUS ${XSDCXX_EXECUTABLE})
-      
+
 # if the include and the program are found then we have it
 IF( XSD_INCLUDE_DIR )
   IF( XSDCXX_EXECUTABLE )

